fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/impl/networking/AbstractNetworkAddon.java

```diff
@@ -64,13 +75,32 @@ public H getHandler(Identifier channel) {
 		}
 	}
 
+	private void assertNotReserved(Identifier channel) {
+		if (this.isReservedChannel(channel)) {
+			throw new IllegalArgumentException(String.format("Cannot register handler for reserved channel with name \"%s\"", channel));
```

Comment:
This IAE may be also thrown if it's for unregistering; consider updating the message.